From weiyi.huang@gmail.com  Mon Oct 20 13:59:28 2008
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Sat, 18 Oct 2008 12:41:57 +0800
Subject: Staging: SLICOSS: remove unused #include <version.h>
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org
Message-ID: <20081018123715.CD78.WEIYI.HUANG@gmail.com>


The file(s) below do not use LINUX_VERSION_CODE nor KERNEL_VERSION.
  drivers/staging/slicoss/slicoss.c

This patch removes the said #include <version.h>.

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/slicoss/slicoss.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/staging/slicoss/slicoss.c
+++ b/drivers/staging/slicoss/slicoss.c
@@ -54,7 +54,6 @@
  *       IS-NIC driver.
  */
 
-#include <linux/version.h>
 
 #define SLIC_DUMP_ENABLED               0
 #define KLUDGE_FOR_4GB_BOUNDARY         1
